src/features/intro/ui/styles.ts

```diff
@@ -112,6 +114,40 @@ export const PageIntroWrapper = styled.section<{
     css`
       animation: ${ANIMATIONS.slideIn} 1s ease-out;
     `}
+  ${MEDIA.mobile} {
+    position: relative;
+    gap: 0px;
+    flex-direction: column;
+    > img {
+      position: absolute;
+      width: 100%;
+      height: 300px;
+      left: ${({ $orderChange, $centerImageOnMobile }) => {
+        if ($centerImageOnMobile) return 'unset';
+        return $orderChange ? '20%' : '-20%';
+      }};
```

Comment:
```suggestion
left: ${({ $orderChange, $centerImageOnMobile }) =>
  $centerImageOnMobile ? 'unset' : $orderChange ? '20%' : '-20%'};
```
오히려 가독성을 떨어트릴 수도 있겠지만... if문을 쓰지 않고 삼항을 중첩해서 쓰는 건 혹시 어떠신가요??

Reply:
가독성을 올린다 -> if
간소화하여 작성한다 -> 삼항
느낌이다 보니 그냥 기존 작성하신 게 더 나을 것 같기도 합니당